subscription-service/src/main/kotlin/com/egm/stellio/subscription/model/GeoQuery.kt

```diff
@@ -3,7 +3,7 @@ package com.egm.stellio.subscription.model
 data class GeoQuery(
     val georel: String,
     val geometry: GeometryType,
-    val coordinates: String,
+    val coordinates: Any,
```

Comment:
Looks good to me since it's mentioned in the spec that the data type is JSON Array or string.
But does it mean that only Point coordinates will be handled as String ?

Reply:
I don't think it only concerns Point coordinates.

I was also wondering if it could break the geo-queries?